src/jdk.jfr/share/conf/jfr/profile.jfc

```diff
@@ -190,6 +190,15 @@
       <setting name="period" control="method-sampling-native-interval">20 ms</setting>
     </event>
 
+    <event name="jdk.CPUTimeSample">
+      <setting name="enabled" control="method-sampling-enabled">false</setting>
+      <setting name="throttle">10ms</setting>
```

Comment:
Can a rate be given here instead of an absolute value? Otherwise, there is a mismatch between the default. jfc and the profile.jfc which seems quite unintuitive.

Reply:
I wanted to give the user the info that both formats are usable (because this is documented nowhere else) and in profiling we usually don't care about having too many events.

Reply:
We do care about having too many events, even profiling.

Reply:
If that is the recalculation from 10 ms according to the adjustments, then yes, this is preferable.

Reply:
It's slightly higher, because I assume that most systems don't use all CPU cores all the time (e.g. GCs, ...).